security/advancedtls/advancedtls_test.go

```diff
@@ -97,47 +97,30 @@ func TestClientServerHandshake(t *testing.T) {
 		// even setting vType to SkipVerification. Clients should at least provide
 		// their own verification logic.
 		{
-			"Client_no_trust_cert_Server_peer_cert",
-			nil,
-			nil,
-			nil,
-			nil,
-			nil,
-			SkipVerification,
-			true,
-			false,
-			false,
-			[]tls.Certificate{serverPeerCert},
-			nil,
-			nil,
-			nil,
-			nil,
-			CertAndHostVerification,
-			true,
+			desc:                       "Client_no_trust_cert_Server_peer_cert",
```

Comment:
You could replace the underscore with spaces in the desc. The `go test` command adds these underscores when it creates subsets (since you are already using `t.Run`).

Reply:
Done